archey/__main__.py

```diff
@@ -78,7 +78,7 @@ def main():
     output = Output()
     for entry in Entries:
         if configuration.get('entries', {}).get(entry.name, True):
-            output.append(entry.name, entry.value().value)
+            output.add_entry(entry.value(name=entry.name))
```

Comment:
I've allowed myself to simplify the instantiation process, tell me what you think about that !

Reply:
Nice! It's definitely a lot less wordy this way, I like it 😄